Source/WindowsServicify/WindowsServicify.Domain/Logger.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace WindowsServicify.Domain;
+
+public static class Logger
+{
+    public static void Log(string message)
+    {
+        string logFileName = DateTime.Now.ToString("yyyy-MM-dd") + ".log";
+        string logDirectory = @"C:\WindowsServicify\Logs";
+
+        if (!Directory.Exists(logDirectory))
+        {
+            Directory.CreateDirectory(logDirectory);
+        }
+
+        string[] logFiles = Directory.GetFiles(logDirectory, "*.log");
```

Comment:
This should be a separate function to be cleaner.

Reply:
Maybe "RemoveOldLogs"

Reply:
Valid 💯